sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Window.java

```diff
@@ -554,6 +556,46 @@ private Assign(WindowingStrategy updatedStrategy) {
           input.getPipeline(), updatedStrategy, input.isBounded());
     }
 
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+
+      builder
```

Comment:
To avoid duplicating the fields of `Window.Bound`, I'd prefer to pass in only the updated windowing strategy. That means that the precise parameters currently being used aren't available from within the transform. Purely for naming purposes, I'd also prefer to not nest Window.Assign in Window.Bound; however, I'm happy to provide a handle back to the enclosing/original transform for an instance of `Window.Assign`, and have the `Window.Assign#populateDisplayData` call back into the original `Window.Bound`

Thoughts?